docs/targets/confluent.md

```diff
@@ -1,107 +1,61 @@
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: *srcname
-
-  template:
-    metadata:
-      labels:
-        app: *srcname
-
-    spec:
-      containers:
-      - name: source
-        image: gcr.io/knativetstenv/awskinesissource-99a4dcc6f886115bdb41983887e528b9:latest
-
-        env:
-
-        # Kinesis stream
-        - name: STREAM
-          value: eventingtst  <-- This will need to be updated.
-
-        - name: AWS_REGION
-          value: us-east-2
-
-        # AWS credentials
-        - name: AWS_ACCESS_KEY_ID
-          value: A... <-- This will need to be updated
-        - name: AWS_SECRET_ACCESS_KEY
-          value: ULw... <-- This will need to be updated
-
-        # Knative system variables
-        - name: NAME
-          value: *srcname
-        - name: NAMESPACE
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.namespace
-        - name: K_LOGGING_CONFIG
-          value: |
-            {
-              "level": "info"
-            }
-        - name: K_METRICS_CONFIG
-          value: |
-            {
-              "domain": "triggermesh.io/sources",
-              "component": "awskinesissource",
-              "configMap": {}
-            }
\ No newline at end of file
+
+## Deploying an Instance of the Target
+
+Open the Bridge creation screen and add a Target of type `Confluent`.
+
+![Adding a Confluent Target](../images/confluent-target/create-bridge-1.png)
+
+In the Target creation form, provide a name to the event Target, and add the following information:
+
+* **Password Secret**: Reference to a [TriggerMesh secret][tm-secret] containing the password associated with the user accessing the Kafka cluster as described in the Prerequisites.
+* **Bootstrap Servers**: Confluent bootstrap servers to connect to. Use `ADD MORE` button to add additional bootstrap servers if needed.
+* **Topic**: Confluent topic to publish events to.
+* **Topic replication**: Number of copies of the topic that should exist in the cluster.
+* **Topic partitions**: Number of partitions for the topic to allow for concurrency.
+* **SASL Mechanism**: Denote how to authenticate against Kafka. Can be one of: `PLAIN` or `GSSAPI`.
+* **Security Protocol**: Denote whether to encrypt the password using SSL/TLS.
+* **Username**: The username to connect to the Kafka cluster as. This field must have a value even if the cluster allows unauthenticated access.
+
+![Confluent Target form](../images/confluent-target/create-bridge-2.png)
+
+After clicking the `Save` button, the console will self-navigate to the Bridge editor. Proceed by adding the remaining components to the Bridge.
+
+![Bridge overview](../images/confluent-target/create-bridge-3.png)
+
+After submitting the bridge, and allowing some configuration time, a green check mark on the main _Bridges_ page indicates that the bridge with a Confluent event Target was successfully created.
+
+![Bridge status](../images/bridge-status-green.png)
```

Comment:
Nitpicky - I  know we use this image in other locations in the docs. Would we like to show the image with the icons, and the bridge name is the same or because we want to reuse it, it does not matter? :)

Reply:
At this point it is more for reuse than anything else.